Package.swift

```diff
@@ -3,6 +3,7 @@ import PackageDescription
 let package = Package(
     name: "Quick",
     exclude: [
+      "Sources/QuickAfterSuiteTests",
```

Comment:
@istx25 
I cannot remember why i excluded this test..
Anyway, @ikesyo san fixed CI failure at #639 , could you review that PR?